src/style.css

```diff
@@ -1,3 +1,75 @@
+:root {
+  --link-color: #000;
+  --link-active-color: #00acdc;
+}
+
+html {
+  font-size: 10px;
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 5rem;
+  align-items: center;
+}
+
+.logo__link {
+  display: block;
+  width: 4rem;
+  height: 4rem;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  font-size: 1.2rem;
+  padding-inline-start: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin: 0 1rem;
+  &:first-child {
+    margin-left: 0;
+  }
+  &:last-child {
+    margin-right: 0;
+  }
```

Comment:
The `&:first-child` and `&:last-child` pseudo-class selectors are being used within a regular CSS file. These are preprocessor features (like in SCSS) and will not work in plain CSS. You need to remove the `&` and write full selectors like `.nav__item:first-child` and `.nav__item:last-child`.